src/components/RadioButton/examples/interactive.jsx

```diff
@@ -0,0 +1,34 @@
+import _ from 'lodash';
+import React from 'react';
+
+import RadioButton from '../RadioButton';
+
+export default React.createClass({
+	getInitialState() {
+		return {
+			isSelected: false
+		};
+	},
+
+	handleSelected(isSelected) {
+		this.setState(_.assign({}, this.state, {
+			isSelected
+		}));
+	},
+
+	render() {
+		return (
+			<ul>
+				<li>
+					<label>Enabled</label>
```

Comment:
Yeah that's how my question around how our labels should work the other day came up -- Dave and Ashley were requesting that labels be included in the examples.